src/arkode/arkode.c

```diff
@@ -3328,15 +3331,28 @@ void arkProcessError(ARKodeMem ark_mem, int error_code, int line,
   char* msg     = (char*)malloc(msglen);
   vsnprintf(msg, msglen, msgfmt, ap);
 
-  if (ark_mem == NULL)
-  {
-    SUNGlobalFallbackErrHandler(line, func, file, msg, error_code);
-  }
-  else
-  {
+  do {
+    if (ark_mem == NULL)
+    {
+      SUNGlobalFallbackErrHandler(line, func, file, msg, error_code);
+      break;
+    }
+
+    if (error_code == ARK_WARNING)
+    {
+#if SUNDIALS_LOGGING_LEVEL >= 1
```

Comment:
This should be `#if SUNDIALS_LOGGING_LEVEL >= SUNDIALS_LOGGING_WARNING`

Reply:
Of course!